biblioteca-de-peliculas-planos/peliculas.md

```diff
@@ -0,0 +1,19 @@
+~Coco~,
```

Comment:
Something that needs to be done to all files is to remove the , on the end of each line !! MUAH MUAHHHH ;)

Reply:
Right away! ;) MUAAAA MUAAAA

Reply:
Done and dusted!